styles.css

```diff
@@ -1008,4 +1007,115 @@ form {
     gap: 4px;
     text-align: center;
   }
+
+  .navbar {
+    position: fixed;
+    width: 100%;
+    z-index: 1;
+    display: flex;
+    flex: 1 1 auto;
+    text-align: center;
+    padding-top: 3em;
+    padding-bottom: 2em;
+    background-color: #fff;
+    justify-content: space-between;
+    box-shadow: 0 0.3rem 2rem rgba(0, 0, 0, 0.1);
+  }
+
+  a.logo {
+    display: flex;
+    flex-direction: row;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 25px;
+    line-height: 20px;
+    color: #6070ff;
+    margin-left: 150px;
+  }
+
+  .navbar nav {
+    position: fixed;
+    top: -120%;
+    width: 100%;
+    height: 100%;
+    background-color: #6070ff;
+    opacity: 0.9;
+  }
+}
+
+.burger {
+  display: none;
+}
+
+@media screen and (max-width: 767px) {
+  .navbar {
+    display: flex;
+    height: 48px;
+    justify-content: space-between;
+    background-color: #fff;
+    align-items: center;
+    position: fixed;
+    z-index: 1;
+    width: 100%;
+  }
+
+  a.logo {
+    display: flex;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    align-self: center;
+    color: #6070ff;
+    text-decoration: none;
```

Comment:
- [ ] [Optional] Please use [backdrop-filter](https://www.w3schools.com/cssref/css3_pr_backdrop-filter.php) property to make the mobile menu background blurry like the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/201704375-4846c608-755f-4d25-ac81-a65237fd8c2c.png) | ![image](https://user-images.githubusercontent.com/68796232/201704406-0f3b9f67-970d-4fae-885b-8379efcfc699.png)  |

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️